tests/testutils/httputils.go

```diff
@@ -44,6 +43,8 @@ func HttpServe(addr string, timeout int) error {
 
 	c := make(chan string)
 	go func() {
+		log.Printf("%v: serving on %v\n", hostname, addr)
```

Comment:
This comment might be misplaced - perhaps I mean https://github.com/coreos/rkt/pull/1608/files#diff-51d007b1d13da6e555f06a639b280874R76 - same sentiment though.